src/utils/isPositiveInteger.m

```diff
@@ -2,14 +2,14 @@
 
 function trueOrFalse = isPositiveInteger(input2check)
     %
-    %
+    % It checks whether the input is a positive integer and report it as ``true`` or ``false``
     %
     % USAGE::
     %
     %   trueOrFalse = isPositiveInteger(input2check)
     %
-    % :param input2check:
-    % :type input2check: column vector
+    % :param input2check: (1xn) The input to check (either a number or ``NaN``)
```

Comment:
```suggestion
    % :param input2check: (nx1) The input to check (either a number or ``NaN``)
```

Maybe double-check that I am correct on that one.

Reply:
I have checked and if there are more than one row it complains

Reply:
OK removing this then.

Reply:
Might be worth making a note of this. And make this more flexible in the future.